build.py

```diff
@@ -54,6 +54,10 @@ def run(cmd):
         f"-DCMAKE_INSTALL_PREFIX={install_dir}/{project} {defines} {shared_cmake} {charmander_header_cmake}")
     run(f"cmake --build {build_dir}/{project} --config Release")
     run(f"cmake --install {build_dir}/{project} --config Release")
+if platform == "windows":
+    run(f"{build_dir}/squirtle/Release/hello.exe")
+elif platform == "linux":
```

Comment:
do we want `otool -L ...` on mac for the same?

Reply:
Sure, but I don't have a Mac to test, can be added later, this is the idea